packages/framework/src/hooks/useEvaluate.ts

```diff
@@ -21,7 +21,7 @@ export const useEvaluate = <T extends Record<string, unknown>>(
       findExpressions(values, [], expressionMap);
       return expressionMap;
     },
-    options?.refreshExpressions ? [values] : [],
+    options?.refreshExpressions ? [values] : [values?.styles],
```

Comment:
Let's leave a comment here - `styles` should be an object that represents a set of style rules. I wonder if this is maybe something we can extend to be more generic and not be special to styles...

Reply:
@sagardspeed2 please leave a comment here